src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
 export interface Figure {
-
+  color: 'red' | 'green' | 'blue',
+  shape: 'triangle' | 'circle' | 'rectangle',
```

Comment:
declare types for color and shape
make shape type enum
```suggestion
  color: Color,
  shape: Shape,
```